packages/odyssey-react-mui/src/ui-shell/SideNav/SideNavItemContent.tsx

```diff
@@ -207,26 +218,30 @@ const StyledNavItemLink = styled(NavItemLink, {
     prop !== "isDisabled" &&
     prop !== "isSelected" &&
     prop !== "odysseyDesignTokens" &&
-    prop !== "sideNavContrastColors",
+    prop !== "sideNavContrastColors" &&
+    prop !== "isDraggedOver",
 })<{
   contextValue: SideNavItemContentContextValue;
   odysseyDesignTokens: DesignTokens;
   sideNavContrastColors: UiShellColors["sideNavContrastColors"];
   isSelected?: boolean;
   isDisabled?: boolean;
+  isDraggedOver: boolean;
```

Comment:
`isDraggedOver` sounds funky. Does this mean it's being hovered while someone is dragging another item? I think there's a term for that: "drop target".

`isDropTarget` might be better. But I think you're want an interim active state like "loading". Like it's currently the drop target rather than it is _a_ drop target.

Can we find a better variable name?

`isActiveDropTarget` would be more correct for what we're saying it is.